addons/l10n_ec/data/account.tax.group.csv

```diff
@@ -0,0 +1,13 @@
+id,code,name,sequence
+ret_ir,ret_ir,Retención Impuesto a la Renta,
+no_ret_ir,no_ret_ir,No sujeto a Retención Impuesto a la Renta,
+vat,vat,IVA diferente de 0%,
+vat0,vat0,IVA 0,
+novat,novat,No objeto de IVA,
+ret_vat_srv,ret_vat_srv,Retención IVA Servicios,
+ret_vat_b,ret_vat_b,Retención IVA Bienes,
+ice,ice,Impuesto a Consumos Especiales,
+isd,isd,Impuesto a la Salida de Capitales,
+iad,iad,Impuesto Aduanas,
+isb,isb,Contribución Super de Bancos,
+comp,comp,Compensaciones Tributarias,
```

Comment:
@Stentor check why do we have tax groups in two different parts everything should be here.

Reply:
Solved in incoming commit.